crates/swift-bridge-ir/src/errors/parse_error.rs

```diff
@@ -68,6 +72,17 @@ pub(crate) enum IdentifiableParseError {
 impl Into<syn::Error> for ParseError {
     fn into(self) -> Error {
         match self {
+            ParseError::ArgsIntoArgNotFound { func, missing_arg } => Error::new_spanned(
+                missing_arg.clone(),
+                format!(
+                    r#"{}: Unknown argument
```

Comment:
Maybe:

format!(`Argument "{}" was not found in the function "{}"'s signature.", missing_arg, func.sig.ident.to_token_stream().to_string());